react/features/mobile/image-cache/middleware.js

```diff
@@ -37,8 +37,13 @@ const _PREFETCH_AVATAR_URLS = false;
 MiddlewareRegistry.register(({ getState }) => next => action => {
     switch (action.type) {
     case APP_WILL_MOUNT:
-    case CONFERENCE_FAILED:
-    case CONFERENCE_LEFT:
+    case SET_LOCATION_URL:
```

Comment:
I just realized how often we're doing this. Why not do it only in APP_WILL_MOUNT? Would it be unrealistic to expect the app to eventually close, before it becomes a problem? As it stands, we are potentially downloading avatars over and over again...

Reply:
OK. Will come in "[RN] Fix a "TypeError: Cannot read property handlers of undefined" in ImageCache".